src/formatDate.js

```diff
@@ -7,8 +7,36 @@
  *
  * @returns {string}
  */
-function formatDate(date, fromFormat, toFormat) {
-  // write code here
+function convertToYYYY(year) {
+  return parseInt(year) < 30 ? '20' + year : '19' + year;
+}
```

Comment:
The `convertToYYYY` function does not handle the case when the year is less than 10. In this case, the result will be a three-digit number, which is not a valid year. You should add leading zeros to the year if it is less than 10.